src/nvim/README.md

```diff
@@ -48,24 +48,29 @@ Requires clang 3.4 or later:
 
     clang --version
 
-Build Nvim with sanitizer instrumentation:
+Build Nvim with sanitizer instrumentation (choose one):
 
     CC=clang make CMAKE_EXTRA_FLAGS="-DCLANG_ASAN_UBSAN=ON"
+    CC=clang make CMAKE_EXTRA_FLAGS="-DCLANG_MSAN=ON"
+    CC=clang make CMAKE_EXTRA_FLAGS="-DCLANG_TSAN=ON"
 
 Create a directory to store logs:
 
     mkdir -p "$HOME/logs"
 
-Enable the sanitizer(s) via these environment variables:
+Configure the sanitizer(s) via these environment variables:
 
     # Change to detect_leaks=1 to detect memory leaks (slower).
     export ASAN_OPTIONS="detect_leaks=0:log_path=$HOME/logs/asan"
-    export ASAN_SYMBOLIZER_PATH=/usr/bin/llvm-symbolizer
-
-    export MSAN_SYMBOLIZER_PATH=/usr/bin/llvm-symbolizer
+    export MSAN_OPTIONS="log_path=${HOME}/logs/tsan"
     export TSAN_OPTIONS="log_path=${HOME}/logs/tsan"
 
-Logs will be written to `${HOME}/logs/*san.PID`.
+Logs will be written to `${HOME}/logs/*san.PID` then.
+
+Ensure that `llvm-symbolizer` is in your `$PATH` to turn virtual addresses
+into file/line locations.
```

Comment:
this should be with the "Requires clang 3.4 or later" (line 47) note.

    Requires clang 3.4 or later, and `llvm-symbolizer` must be in `$PATH`.